server/src/main/java/org/spine3/server/users/CurrentTenant.java

```diff
@@ -22,11 +22,12 @@
 
 import org.spine3.SPI;
 import org.spine3.users.TenantId;
-import org.spine3.validate.Validate;
 
 import javax.annotation.Nullable;
 
+import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
+import static org.spine3.validate.Validate.isNotDefault;
 
 /**
  * This class allows to set a tenant when handling a command or a query
```

Comment:
Maybe we should provide some links for those who are not familiar with multitenant architecture?